pistrap

```diff
@@ -159,6 +174,14 @@ while [ -n "${1+xxx}" ]; do
             RASPI_IMAGE_URL="$(find_latest)"
             RASPI_IMAGE_NAME="$(basename "${RASPI_IMAGE_URL}")"
             ;;
+        -400|--pi400)
+            if [ -n "${RASPI_IMAGE_NAME_400}" ]; then
+                usage 1 "RaspberryPi image already specified. Please do not combine -l and -i options. Duh."
+            fi
+
+            RASPI_IMAGE_URL_400="$(find_latest_400)"
+            RASPI_IMAGE_NAME_400="$(basename "${RASPI_IMAGE_URL_400}")"
+            ;;
```

Comment:
Do not use new variable names for the new image, just change the variable contents and adapt the code to accommodate for this change.